cmd/eksctl-anywhere/cmd/deletecluster.go

```diff
@@ -45,23 +41,11 @@ var deleteClusterCmd = &cobra.Command{
 	},
 }
 
-func preRunDeleteCluster(cmd *cobra.Command, args []string) error {
-	cmd.Flags().VisitAll(func(flag *pflag.Flag) {
-		err := viper.BindPFlag(flag.Name, flag)
-		if err != nil {
-			log.Fatalf("Error initializing flags: %v", err)
-		}
-	})
-	return nil
-}
-
 func init() {
 	deleteCmd.AddCommand(deleteClusterCmd)
+	addClusterFlags(deleteClusterCmd, &dc.clusterOptions, "filename", "install-packages")
```

Comment:
we are adding flags here that wasn't there before, like `install-packages`

Reply:
I was using the last "filename" and "install-packages" argument as an override that will cause addClusterFlags to exclude the `install-packages` and `filename` flag so they can be added in deletecluster.go. 

I realize it doesn't seem very intuitive to list out the flags that should NOT be included, but I thought it'd be more concise than including all the ones that should. Any suggestions on getting this to make more sense?

Reply:
Decided to not touch the flag grouping in this PR. I can make a separate PR as needed if it helps to group the flags.